modules/sync/syncs/urlsynchronization.cpp

```diff
@@ -46,105 +48,78 @@ namespace {
     constexpr const char* KeyFilename = "Filename";
 
     constexpr const char* TempSuffix = ".tmp";
+
+    struct [[codegen::Dictionary(UrlSynchronization)]] Parameters {
+        // The URL or urls from where the files are downloaded. If multiple URLs are
+        // provided, all files will be downloaded to the same directory
+        std::variant<std::string, std::vector<std::string>> url;
+
+        // This optional identifier will be part of the used folder structure and, if 
+        // provided, can be used to manually find the downloaded folder in the
+        // synchronization folder. If this value is not specified, 'UseHash' has to be set
+        // to 'true'
+        std::optional<std::string> identifier;
+
+        // If this value is set to 'true' and it is not overwritten by the global
+        //settings, the file(s) pointed to by this URLSynchronization will always be 
+        // downloaded, thus overwriting the local files. This is useful for files that are 
+        // updated regularly remotely and should be fetch at every startup
+        std::optional<bool> override;
```

Comment:
Not so nice that this variable has a name that corresponds to a C++ specifier. Don't know if it's actually going to be a problem though

Reply:
Maybe rename to `_forceOverride`, which is the name for the property, and use the `codegen::Key` thing?

Reply:
Yea;  good point.  It's not a problem compilation wise, but it confuses the syntax highlighter.